react/ecmascript/01 - spread/index.js

```diff
@@ -1,7 +1,54 @@
 import isArray from './utils';
 
-export function min(numbers) {
-  return Math.min(...numbers);
+export function min(...numbers) {
+  if (numbers.length !== 0) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+    if (numbers.length === 1) {
+      return Math.min(numbers);
+    }
+    const array = [];
+    for (let i = 0; i < numbers.length; i++) {
+      array.push(numbers[i]);
+    }
+    return Math.min(...array);
+  }
+  return undefined;
 }
 
-export function copy() {}
+export function copy(numbers) {
```

Comment:
You are supposed to use the spread operator for this.
Keep in mind you can use it inside arrays and objects to "copy" what's inside them on some other object or array.